lib/mavenlink/integrated_resource.rb

```diff
@@ -0,0 +1,24 @@
+module Mavenlink
+  class IntegratedResource < Model
+    def save
+      if valid?
+        reload(create_or_update(attributes_for_create_or_update))
+        true
```

Comment:
Not sure. We've mirrored exactly what the default `save` does.

Check out https://github.com/mavenlink/mavenlink_gem/blob/08395b2403dec59f7cc8960f9317bc8c5ad90d30/lib/mavenlink/model.rb#L150